src/zope/i18n/locales/inheritance.py

```diff
@@ -226,6 +225,8 @@ def items(self):
     def keys(self):
         return list(self._make_reified_inherited_dict().keys())
 
-    def value(self):
-        # XXX: Was this meant to override values() and is just a typo?
+    def values(self):
         return list(self._make_reified_inherited_dict().values())
+
+    # Preserve the old name for compatibility
+    value = values
```

Comment:
Is it hard to add a deprecation warning on the use of this?

Reply:
We don't have a dependency on zope.deprecation, so we would either need to add one or write the code manually. That would have to be tested, which either means in the docstring (thus documenting it in perpetuity) or a whole new test file.  That all seemed like overkill to me.

Maybe we should just drop the old name entirely, on the theory that no one was actually using it?

Reply:
`zope.deprecation` is a lightweight dependency, so I added that.